.rubocop.yml

```diff
@@ -10,3 +10,6 @@ Metrics/BlockLength:
   Exclude:
     - 'features/**/*.rb'
     - 'spec/**/*'
+
+Style/AndOr:
+  EnforcedStyle: conditionals
```

Comment:
This is the expected behaviour. See our styleguide: https://github.com/alphagov/govuk-lint/blob/master/configs/rubocop/gds-ruby-styleguide.yml#L127-L130

Reply:
There's some discussion about why this rule is not quite right for `and return` situations here:
https://github.com/rubocop-hq/rubocop/issues/5550
but, @barrucadu has suggested another way round in this case anyway 😄